xbmc/pvr/recordings/PVRRecording.h

```diff
@@ -82,8 +84,23 @@ namespace PVR
     std::string   m_strFanartPath;    /*!< fanart path */
     unsigned      m_iRecordingId;     /*!< id that won't change while xbmc is running */
 
+  private:
```

Comment:
Well ..... no. Don't get how this is related to compiler optimizations. This is about utilizing the compiler to help you preventing programming errors in the first place by restricting access privileges. And creating copies leads to all the problems and "find the original instance belonging to a copy" crap and many subtile bugs realated to the insane design the code now is suffering from. Don't see any drawbacks here, Please explain to me why this approach should be no good.